packages/@uppy/webdav/src/locale.ts

```diff
@@ -0,0 +1,7 @@
+export default {
+  strings: {
+    pluginNameWebdav: 'WebDAV',
+    authenticate: 'Connect',
+    pluginWebdavInputLabel: 'WebDAV, ownCloud, or Nextcloud URL',
+  },
```

Comment:
The old string was more specific tho. A "public link" is a specific concept in ownCloud/Nextcloud

Reply:
and just the url to the root of the instance doesn't make sense for the simple auth provider (only for the oauth one we're not adding)